src/core/init/stall_avoider.ts

```diff
@@ -143,6 +171,24 @@ export default function StallAvoider(
                  value: null };
       }
 
+      if (tick.seeking) {
+        lastSeekingPosition = tick.position;
```

Comment:
Is there a risk here that the position may change to the past while the 'seeking' attribute is still true ?

Reply:
I don't know but that's not what we observed in our tests

Reply:
Ok but then, shouldn't it be specific to Tizen?

Reply:
Done